README.md

```diff
@@ -1 +1,69 @@
-# metrics-webapp
\ No newline at end of file
+# Space-Travelers-Hub
+- In this group task, we were working with the real live data from the SpaceX API. Our task was to build a web application for a company that provides commercial and scientific space travel services. The application allows users to book rockets, dragons and join selected space missions.
+
+![main_Page](https://github.com/zemola/metrics-webapp/blob/feature/basic-structure/src/images/main.png)
+![Detail_Page](https://github.com/zemola/metrics-webapp/blob/feature/basic-structure/src/images/detail.png)
+
+## Built With
+
+- React
+- Redux
+- JavaScript
+- CSS
+- VS code
+
+## Live Demo
+
+> Project live demo on [Netlify](https://coviid-webapp.netlify.app/)
+
+## PROJECT PRESENTATION
+
+> Project live demo on [LOOM](https://www.loom.com/share/684ec67477ea4b3995e448a328a6ff30)
+
+# Getting Started
+## Prerequisites:
+
+
+- Create an account on github
+
+- Install gitbash or use git on the command line.
+
+- Use an IDE of your preference such as VScode.
+
+## Steps to follow:
+
+- Choose a folder in your local machine where you want this repository to be copied
+
+- Clone this [repository](https://github.com/zemola/metrics-webapp.git) to your locaL machine with `git clone` command.
+It should look similar to this git clone https://github.com/smithj/azure-docs.git Enter your github credentials if prompted.
+
+- Navigate to react-group-project directory.
+
+- On the command line, at the project's root, run 'npm install' to install app dependencies.
+
+- Next, run 'npm start' which will run the app in the development mode.
+
+
+## Author
+
+
+- [GitHub](https://github.com/zemola)
+- [LinkedIn](https://www.linkedin.com/in/olatunjiazeem/)
+- [Twitter](https://twitter.com/zemolat)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/zemola/metrics-webapp/issues)).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+Thanks to my teammates, learning and coding partners.
```

Comment:
As required, Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).